CHANGELOG.rst

```diff
@@ -36,6 +36,8 @@ Features and Enhancements
   definitions in a Cortex and to add extended extended model definitions to
   a Cortex in bulk.
   (`#3252 <https://github.com/vertexproject/synapse/pull/3252>`_)
+- Add ``inet:ipv5`` to the list of types identified with scrape APIs.
```

Comment:
```suggestion
- Add ``inet:ipv6`` to the list of types identified with scrape APIs.
```

Reply:
Do you want to mention that this will also detect `inet:server`s with ipv6 addresses?